src/components/community/community-button.js

```diff
@@ -0,0 +1,28 @@
+import '@brightspace-ui/core/components/button/button-subtle.js';
+import { CommunityBase } from './community-base.js';
+import { html } from 'lit';
+
+class CommunityLink extends CommunityBase {
+
+	static get properties() {
+		return {
+			...CommunityBase.properties,
```

Comment:
I'd suggest doing `CommunityBase` as a Lit mixin instead -- that way these inherited classes don't need to add its properties to theirs.